index.js

```diff
@@ -0,0 +1,330 @@
+    usedCellArray.push(event.target);
+    checkWin();
+
+    let win = checkWin();
+
+    if (win) {
+        if (currentPlayer === 'x') {
+            status.textContent = `${capitalize(playerOne)} wins!`
+        } else {
+            status.textContent = `${capitalize(playerTwo)} wins!`
+        }
+        stopPlay(cellArray)
+        clearInterval(interval);
+        clockCount = 0;
+        startBtn.disabled = true;
+        onePlayer.disabled = false;
+        twoPlayer.disabled = false;
+        playerOneName.disabled = false;
+        playerTwoName.disabled = false;
+    } else if (parseInt(usedCellArray.length) === 9) {
+        status.textContent = `It's a draw!`;
+        onePlayer.disabled = false;
+        twoPlayer.disabled = false;
+        playerOneName.disabled = false;
+        playerTwoName.disabled = false;
+        stopPlay(cellArray)
+        clearInterval(interval);
+        clockCount = 0;
+    } else {
+        if (gameMode === 'onePlayer') {
+            if (currentPlayer === 'x') {
+                stopPlay(cellArray);
+                currentPlayer = 'o';
+                status.textContent = `${capitalize(playerTwo)}'s turn!`
+                let randomCell = cellArray[Math.floor(Math.random() * 9)];
+                while (randomCell.textContent !== '' && randomCell) {
+                    randomCell = cellArray[Math.floor(Math.random() * 9)];
+                }
+                randomCell.removeEventListener('click', alreadyClicked)
+                randomCell.addEventListener('click', clicked);
+                setTimeout(() => {
+                    randomCell.click()
+                }, 1000);
+                cellArray.forEach(function (cell) {
+                    cell.addEventListener('click', clicked);
+                })
+            } else {
+                currentPlayer = 'x';
+                status.textContent = `${capitalize(playerOne)}'s turn!`;
+                cellArray.forEach(function (cell) {
+                    cell.addEventListener('click', alreadyClicked);
+                })
+            }
+        } else if (gameMode === 'twoPlayer') {
+            if (currentPlayer === 'x') {
+                currentPlayer = 'o';
+                status.textContent = `${capitalize(playerTwo)}'s turn!`
+            } else {
+                currentPlayer = 'x';
+                status.textContent = `${capitalize(playerOne)}'s turn!`;
+            }
+        }
+    }
+}
+
+// function that tells the player to select an empty cell if they've selected one that's 
+// already in use
+function alreadyClicked() {
+    if (gameMode === 'onePlayer' && currentPlayer === 'o') {
+        let randomCell = cellArray[Math.floor(Math.random() * cellArray.length)];
+        randomCell.click();
+    }
+    status.textContent = 'Please select an empty cell.';
+}
+
+startBtn.addEventListener('click', () => {
+    start();
+    onePlayer.disabled = true;
+    twoPlayer.disabled = true;
+})
```

Comment:
If you wanted to keep your mouse over events as JS you could add them to this button for a better functionality